list.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>List</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet" type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="cards-container">
+      <div class="mini-card">
+        <img class="mini-cover-image" src="img/book-cover.png">
```

Comment:
make sure to always add an alt to images: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img